homeassistant/components/lutron/cover.py

```diff
@@ -33,7 +35,7 @@ async def async_setup_entry(
     entry_data: LutronData = hass.data[DOMAIN][config_entry.entry_id]
     async_add_entities(
         [
-            LutronCover(area_name, device, entry_data.covers)
+            LutronCover(area_name, device, entry_data.client)
```

Comment:
This has been introduced in 2024.2. I could make a separate PR for this small thing but it would only increase overhead, hence I just put it in here (as I'm expecting this will merged quite quick)

Reply:
👍🏻  Didn't know that hasn't made it to production yet.